smack-core/src/main/java/org/jivesoftware/smack/packet/StreamOpen.java

```diff
@@ -17,49 +17,12 @@
 
 package org.jivesoftware.smack.packet;
 
-import org.jivesoftware.smack.util.StringUtils;
 import org.jivesoftware.smack.util.XmlStringBuilder;
 
 /**
  * The stream open <b>tag</b>.
  */
-public class StreamOpen implements Nonza {
-
-    public static final String ELEMENT = "stream:stream";
-
-    public static final String CLIENT_NAMESPACE = "jabber:client";
-    public static final String SERVER_NAMESPACE = "jabber:server";
-
-    /**
-     * RFC 6120 § 4.7.5.
-     */
-    public static final String VERSION = "1.0";
-
-    /**
-     * RFC 6120 § 4.7.1.
-     */
-    private final String from;
-
-    /**
-     * RFC 6120 § 4.7.2.
-     */
-    private final String to;
-
-    /**
-     * RFC 6120 § 4.7.3.
-     */
-    private final String id;
-
-    /**
-     * RFC 6120 § 4.7.4.
-     */
-    private final String lang;
-
-    /**
-     * RFC 6120 § 4.8.2.
-     */
-    private final String contentNamespace;
-
+public class StreamOpen extends AbstractStreamOpen {
```

Comment:
This class should become `final`.

Reply:
@Flowdalic , Aren't we going to inherit this class into `WebsocketOpen` ?